### DISCUSSION TITLE
Dream Router, KPN, stb-nmc-400 error

### DISCUSSION THREAD

**Discussion by RonaldBlom** [fabianishere/udm-iptv#147] [Q&A] [answered]
Hi, I just migrated from USG 3-P to Dream Router. Before I was having a config_gateway.json file in place to get iptv and all working. That was pretty stable lately.

Now all looks good. But getting stb-nmc-400 error constantly after changing channel or resetting kpn tvboxes (VIP5202).

Please share the following output with the developers:
=== Configuration ===
WAN Interface: eth4
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 213.75.0.0/16  217.166.0.0/16
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
70: iptv@eth4: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.230.93.66/22 brd 10.230.95.255 scope global iptv
       valid_lft forever preferred_lft forever
10.230.92.0/22 proto kernel scope link src 10.230.93.66
213.75.112.0/21 via 10.230.92.1 metric 270
=== Service Logs ===
Jul 22 12:36:04 UDRBlomstraal systemd[1]: Started IPTV support for the UniFi Dream Machine.
Jul 22 12:36:04 UDRBlomstraal udm-iptv[32186]: Device iptv already exists.. Deleting device
Jul 22 12:36:04 UDRBlomstraal udm-iptv[32186]: Obtaining IP address for VLAN interface
Jul 22 12:36:04 UDRBlomstraal udm-iptv[32186]: udhcpc (v1.22.1) started
Jul 22 12:36:04 UDRBlomstraal udm-iptv[32186]: Sending discover...
Jul 22 12:36:05 UDRBlomstraal udm-iptv[32186]: Sending select for 10.230.93.66...
Jul 22 12:36:05 UDRBlomstraal udm-iptv[32186]: Lease of 10.230.93.66 obtained, lease time 3600
Jul 22 12:36:05 UDRBlomstraal udm-iptv[32186]: NATing IPTV network ranges (if necessary)
Jul 22 12:36:05 UDRBlomstraal udm-iptv[32186]: Setting up igmpproxy
Jul 22 12:36:05 UDRBlomstraal udm-iptv[32186]: Starting igmpproxy

hope you can assist

**Comment by fabianishere** [comment, marked as answer]
Make sure you have a stable connection from your IPTV receiver to the UDR. I recommend placing it on a separate VLAN if possible.

**Reply by RonaldBlom** [reply]
Hi Fabian, thanks for quick reply. It is in seperate vlan. It is fixed allready. During install I thought I selected the vlan, but I did not see I had to select the line with the spacebar to actually select it. So after removin the install and reinstalling it correctly all is working.

Thanks for your effort. Have a nice day.
